docs/source/user/fast.farm/FFarmTheory.rst

```diff
@@ -1291,6 +1292,135 @@ the solution of the momentum equation),
 :math:`V_{r_{n_p,n_r-1}}^\text{Wake}\left[ n+1 \right]` for
 :math:`1\le n_r\le N_r-1`. [10]_
 
+
+
+.. _FF:WAT:
+
+Wake-Added Turbulence (WAT)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Wake-added turbulence is the additional small-scale turbulence generated
+from the turbulent mixing in the wake. 
+It is modeled by scaling up a background (undisturbed) turbulence.
+
+
+The theory for WAT will be presented in more details in the future as part of a journal publication.
+
+
+The basic principle for the wake-added turbulence is illustrated in :numref:`FF:WATSketch`. 
+
+.. figure:: Pictures/FFWakeAddedTurbBoxCoord.svg
+   :alt: Wake Added turbulence
+   :name: FF:WATSketch
+   :width: 100%
+   :align: center
+
+   Wake-added turbulence
+
+A scaling factor is computed at each wake plane, it is multiplied with a unit turbulence box
+and added to the quasi steady wake to form the final wake with wake-added turbulence. 
+In this implementation, the scaling factors are computed in the meandering
+frame, but assembled with the “global” unit turbulence box in the global
+frame. More details follow.
+
+
+**Scaling factor**
+
+The scaling factor, expressed in terms of the wake deficit :math:`V_x^{Wake}`, is determined at each wake plane as:
+
+.. math::
+
+   \begin{aligned}
+      k_{} (x,y,z) = 
+          \frac{k_\text{def} }{\overline{U}} \left| V_x^{Wake}(x,y,z) \right|
+        + \frac{k_\text{grad}D}{2\overline{U}} \left[\left|{\frac{\partial {V_x^{Wake}(x,y,z)}}{\partial r}}\right| +  \left|{\frac{1}{r}\frac{\partial {V_x^{Wake}(x,y,z)}}{\partial \theta}}\right|  \right]  
+        \end{aligned}
+
+where :math:`D` is a reference diameter, and :math:`\bar{U}` is the mean velocity taken as the filtered velocity at the turbine location normal to the rotor disk.
+The coordinates :math:`x,y,z` and :math:`r,\theta` are taken in the meandering frame of reference.
```

Comment:
Suggest clarifying that the last term (involving r,\theta) is zero for the polar wake and nonzero for the curled wake.